### DISCUSSION TITLE
Disabling Mentions

### DISCUSSION THREAD

**Discussion by FerasK0** [robertodoering/harpy#607] [Q&A] [answered]
Hi!
Is there any explanation or notes for the disabling of mentions?
I searched and found nothing useful here and i have no clue.
But my best guess would be that the mentions are causing some bugs or interferes with some features somehow?

Thanks in advance :)

**Comment by robertodoering** [comment, marked as answer]
Twitter restricts API access if users of your app spam @mentions. Hence the issues in the past 2-3 weeks where we had the API access revoked and had to update the app with a new key multiple times.

This isn't a new thing apparently and other developers have had hit the same issue [[1]](https://twittercommunity.com/t/api-access-restricted/164789) [[2]](https://twittercommunity.com/t/api-restricted-from-doing-mentions/165666) [[3]](https://twittercommunity.com/t/api-restricted-from-doing-mentions/165666),  but it's the first time it has happened to harpy.


Basically we will have to add a limitation on how often a user can @mention other users in their tweets so that our API access doesn't get automatically revoked.
I'll add this with the next big update that I'm currently working on but until then @mentions are completely disabled.

**Reply by dnestico** [reply]
Such a shame twitter puts these restrictions on their API, meanwhile Facebook/Instagram is somehow 10x worse so I'll give them that.

**Reply by FerasK0** [reply]
Completely justified and waiting for that massive update, but how much is the limit?
Like 5 mentions per hour or what?

**Reply by robertodoering** [reply]
As it usually seems to be the case with the Twitter API, these things are incredibly vague or simply unspecified. [This thread](https://twittercommunity.com/t/unsolicited-mentions-account-restricted-from-performing-write-actions/69576) on the Twitter developer forum was the best information for this issue that I could find.
